src/components/carrousel/CarousselPane.tsx

```diff
@@ -0,0 +1,54 @@
+interface Props {
+  children: JSX.Element[] | JSX.Element | string;
+  id: number;
+  numPanes: number;
+  activePane: number;
+}
+
+export const mod = (k: number, n: number): number => {
```

Comment:
This should be moved to a util file or something since its being imported elsewhere